data/korath outfits.txt

```diff
@@ -137,6 +137,18 @@ outfit "Systems Core (Large)"
 	"energy capacity" 5200
 	description `This massive bank of computers, shield generators, and repair stations holds everything needed to keep the very largest of Korath ships in working condition.`
 
+
+outfit "Fuel Processor"
+        category "Systems"
```

Comment:
You have spaces before these attributes instead of tabs.

Reply:
Oops. Replaced them.